lib/stripe/issuing/card_details.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Stripe.Issuing.CardDetails do
+  @moduledoc """
+  Work with Stripe Issuing card details.
+
+  You can:
+
+  - Retrieve card details
+
+  Stripe API reference: https://stripe.com/docs/api/issuing/cards/retrieve_details
+  """
+
+  use Stripe.Entity
+  import Stripe.Request
+
+  @type t :: %__MODULE__{
+          card: Stripe.Issuing.Card.t(),
+          object: String.t(),
+          cvc: String.t(),
+          exp_month: String.t(),
+          exp_year: String.t(),
+          number: String.t()
+        }
+
+  defstruct [
+    :card,
+    :object,
+    :cvc,
+    :exp_month,
+    :exp_year,
+    :number
+  ]
+
+  @plural_endpoint "issuing/cards"
+
+  @doc """
+  Retrieve card details.
+  """
+  @spec retrieve(Stripe.id() | Stripe.Issuing.Card.t(), Stripe.options()) ::
+          {:ok, t} | {:error, Stripe.Error.t()}
+  def retrieve(id, opts \\ []) do
```

Comment:
Is this the same as the retrieve endpoint in `Stripe.Issuing.Card`?  Is this module necessary?

Reply:
CardDetails is a nested resource used to retrieve the full sensitive details for an issued card. I treated it like a separate and distinct resource because the struct is very different from the Card resource used to manage issued cards.

Reply:
I see.  In their example response, it looks like the response object contains most of what is in the `Issuing.Card` struct.  Am I missing something?

https://stripe.com/docs/api/issuing/cards/retrieve_details

Reply:
There is some overlap, but the CardDetails has a Card child object and then the sensitive attributes for :cvc and :number. I'm open to suggestions if it makes more sense to define the CardDetails struct and the `retrive_card_details` function in the Card module. I see them as distinct structs but concede that it might be unnecessary to have a separate module.

Reply:
@jcartwright Well I guess I am curious - if we limit the attributes to just these few, then won't we be discarding all of them that may come in the response?

Reply:
All of the common attributes are in the CardDetails `:card` child element, so they are not discarded but simply nested.